src/diff/children.js

```diff
@@ -111,6 +111,7 @@ export function diffChildren(
 			oldVNode = oldChildren[i];
 			if (oldVNode && oldVNode.key == null && oldVNode._dom) {
 				if (oldVNode._dom == oldDom) {
+					oldVNode._parent = oldParentVNode;
```

Comment:
I am currently trying to figure out why this part matters but it looks like `oldVNode._parent` refers to the new version of the parent-vnode here